bowling/file.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <map>
+#include <string>
+#include <vector>
+
+class File {

```

Comment:
it rather represents an InputProcessor, not File. It contains a path to file as a member. Maybe it will be even better to keep ifstream, not filename as a string?